src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSpec.scala

```diff
@@ -42,7 +42,7 @@ class ClusterMonitorSpec extends TestKit(ActorSystem("leonardotest")) with FlatS
                                             dataprocInfo = makeDataprocInfo(1).copy(hostIp = None),
                                             status = ClusterStatus.Creating,
                                             userJupyterExtensionConfig = Some(userExtConfig),
-                                            stopAfterCreation = true)
```

Comment:
Was causing NPEs in tests which didn't handle `stopAfterCreation`. It only needs to be set in the tests which validates stopping after creation.